frontend/src/libs/import-export/undefinedToNull.js

```diff
@@ -0,0 +1,33 @@
+/**
+ * Util functions to turn undefined to null for json
```

Comment:
This is confusing, since JSON doesn't have `undefined`. It is already the case that `JSON.stringify` converts `undefined` to `null`. So, why do we need this function?

Also, let's make new files TypeScript by default.

Reply:
Removed this redundant function and added a comment explaining why it is not needed.